### PR TITLE
table td wrap

### PR DESCRIPTION
**Pull request by silverailscolo**
Wrap long table cell text blocks. Plus 1 html fix

### CODE REVIEW

File: community/clinics/index.shtml [hunk 1]
```diff
@@ -22,26 +22,37 @@
     
     <p>Upcoming JMRI Clinics as submitted or found on the web:
 
-    July 3-10, 2016 - Indianapolis, IN
```
Comment (danielb987) [inline community/clinics/index.shtml:25]: Not related to this PR, but these clinics are not upcoming. It seems reasonable to move them to the next section: Past clinics.

**Review by danielb987 — APPROVED** [review]

**Comment by danielb987** [comment]
@silverailscolo 
This PR is merged and it seems it has gone wrong. See this page:
https://www.jmri.org/help/en/html/doc/Technical/JVMCapabilities.shtml

The table is not on the right of the sidebar but below the sidebar. So my first thought was that there was no content on the page. And my screen is 1920x1080 pixels.

**Comment by danielb987** [comment]
@silverailscolo 
The problem might be worse whan I first thought. Look at https://www.jmri.org/. I only see the sidebar. I have to scroll down to see the main content. On a 1920x1080 screen.

**Comment by danielb987** [comment]
@silverailscolo 
When you look at this, I suggest that you merge the PR immediately once you have created it. For JMRI/JMRI we have the requirements that someone else reviews the PR and that 24 hours have passed before merge. But we don't have these requirements for JMRI/website.

When it comes to small changes, like this, I think it's better if you merge the PRs at once. It gives you the possibility to see the changes soon so that problems can be fixed. And it's easier for everyone to see problems when the changes of the web site is live.

And changes to the web site doesn't affect the source code so even big problems on the web site is small compared to potential problems to the Java source code.

**Thank you for your work on the web site!**

**Comment by silverailscolo** [comment]
Thanks, I blew css with an unclosed comment; fixed since.
Table wrap fix is in https://github.com/JMRI/JMRI/pull/9874 on the JMRI repo.

**Pull request merged**